includes/definitions/discovery/apc.yaml

```diff
@@ -238,6 +277,13 @@ modules:
                     index: 'airIRRP100UnitStatusHumidifierCurrent.{{ $index }}'
                     descr: Humidifier
                     divisor: 10
+                -
+                    oid: rPDU2PhaseStatusTable
+                    value: rPDU2PhaseStatusCurrent
```

Comment:
This might be redundant as we already fetch it in includes/discovery/sensors/current/apc.inc.php?